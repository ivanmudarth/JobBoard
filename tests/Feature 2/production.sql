-- Feature 2

-- users dhruvs(id = 2) and ivanm(id = 3) both add jobs to their shortlist 
INSERT INTO Shortlist VALUES (2, 54)
INSERT INTO Shortlist VALUES (3, 372)
INSERT INTO Shortlist VALUES (3, 524)

SELECT * FROM Shortlist

-- view ivanm's shortlist 
SELECT DISTINCT job_posting_id FROM Shortlist WHERE user_id = 3

-- delete job_posting_id of 524 from ivanm's shortlist 
DELETE FROM Shortlist WHERE user_id = 3 AND job_posting_id = 524

-- view ivanm's shortlist again after deletion 
SELECT DISTINCT job_posting_id FROM Shortlist WHERE user_id = 3