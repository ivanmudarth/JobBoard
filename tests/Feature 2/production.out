Execute:
> SELECT * FROM Shortlist

+ ------------ + ------------------- +
| user_id      | job_posting_id      |
+ ------------ + ------------------- +
| 2            | 54                  |
| 3            | 372                 |
| 3            | 524                 |
+ ------------ + ------------------- +
3 rows


Execute:
> SELECT DISTINCT job_posting_id FROM Shortlist WHERE user_id = 3

+ ------------------- +
| job_posting_id      |
+ ------------------- +
| 372                 |
| 524                 |
+ ------------------- +
2 rows


// After deletion: 
Execute:
> SELECT DISTINCT job_posting_id FROM Shortlist WHERE user_id = 3

+ ------------------- +
| job_posting_id      |
+ ------------------- +
| 372                 |
+ ------------------- +
1 rows
